packages/relay/src/lib/clients/sdkClient.ts

```diff
@@ -81,8 +76,19 @@ export class SDKClient {
             .execute(this.clientMain);
     }
 
+    async getContractBalance(contract: string): Promise<AccountBalance> {
+        return (new AccountBalanceQuery()
+            .setContractId(ContractId.fromString(contract)))
+            .execute(this.clientMain);
+    }
+
+    async getContractBalanceInWeiBar(account: string): Promise<BigNumber> {
```

Comment:
```suggestion
    async getContractBalanceInTinyBar(account: string): Promise<BigNumber> {
```

Reply:
This should stay as it is. The function `toTinyBar` was named incorrectly.